### PR TITLE
Added my submission

### PR DESCRIPTION
**Pull request by jiachenyee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ To add your own project below, just [edit](https://github.com/wwdc/2021/edit/mas
 |[Filip Kania](https://github.com/filipkania)|[GitHub](https://github.com/filipkania/getout.)||SceneKit, AVFoundation, AppKit|Submitted|
 |[Furkan Hancı](https://github.com/Furkanus)|[Github](https://github.com/Furkanus/BioShine)||SwiftUI|Submitted|
 |[Haotian Zheng](https://fincher.im/)|[GitHub](https://github.com/JustinFincher/WWDC2021ScholarshipProject)|[YouTube](https://www.youtube.com/watch?v=AT6XDYx_aRg)|ARKit, SceneKit, MetalKit, SwiftUI|Submitted|
+|[Jia Chen](https://github.com/jiachenyee)|[GitHub](https://github.com/jiachenyee/wwdc21explorer)||SceneKit, UIKit, SwiftUI, Natural Language, Playground Support|Submitted|
```
Comment (thisIsTheFoxe) [inline README.md:29]:
```suggestion
|[Jia Chen](https://github.com/jiachenyee)|[GitHub](https://github.com/jiachenyee/wwdc21explorer)||SceneKit, UIKit, SwiftUI, Natural Language|Submitted|
```

**Review by thisIsTheFoxe — CHANGES_REQUESTED** [review]
Hi, thanks for your contribution. Could you just remove Playground Support, since everyone has to use it in some way. Thanks :)

**Comment by jiachenyee** [comment]
Ok, I've fixed the issue

**Review by thisIsTheFoxe — APPROVED** [review]
Thanks again and best of luck.. cya @ dubDub

**Pull request merged**